de-lib/src/test/java/org/iplantc/de/preferences/client/presenter/PreferencesPresenterImplTest.java

```diff
@@ -46,8 +58,8 @@ public void setUp() {
         when(viewMock.getValue()).thenReturn(userSettingsMock);
 
         uut = new PreferencesPresenterImpl(viewMock,
-                                           serviceFacadeMock,
-                                           appearanceMock);
+                                           serviceFacadeMock, oauthServiceFacadeMock,
+                                           appearanceMock, constantsMock);
```

Comment:
Also minor, but each parameter should probably be on its own line.

Reply:
👍